test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -345,36 +358,34 @@ private static string BasicAuth(string? token)
             return Convert.ToBase64String(Encoding.UTF8.GetBytes($"user:{token}"));
         }
 
-        private static Task SendPullRequestComments()
+        private static Task SendPullRequestComments(bool crashed = false, string message = "")
```

Comment:
nit: one arg is enough?
```suggestion
        private static Task SendPullRequestComments(string? crashMessage = null)
```